policies/stable-release-updates.md

```diff
@@ -68,8 +68,8 @@ An **end-user documentation** is **not**:
 - comments in the code (including public header files)
 - internal documentation (doc/internal)
 
-Changes allowed in stable releases
-----------------------------------
+Changes allowed in stable releases and pre-releases after the first beta
+------------------------------------------------------------------------
 
 No API or ABI breaking changes are allowed in a minor or patch release.
```

Comment:
Nit-pick: This sentence could use a slight adjustment

Reply:
The vote already started.

Reply:
So an editorial follow-up PR.  Check